imports/plugins/core/accounts/client/components/permissionsList.js

```diff
@@ -100,18 +100,16 @@ class PermissionsList extends Component {
   renderPermissions(permissions) {
     const jsx = [];
     permissions.forEach((permission, key) => {
-      jsx.push(
-        <div className="permission-item" key={`${permission.name}-${key}`}>
-          <Components.ListItem
-            label={permission.label}
-            actionType="switch"
-            switchOn={this.checked(permission.name)}
-            switchName={permission.name}
-            onSwitchChange={this.togglePermission(permission)}
-          />
-          {this.renderSubPermissions(permission)}
-        </div>
-      );
+      jsx.push(<div className="permission-item" key={`${permission.name}-${key}`}>
```

Comment:
To me, this is more confusing having the `div` precedeced by the `jsx.push` on the same line.